changelogs/fragments/64751-fix-wrong-promt-len-calc-in-ansible-console.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - fix wrong command line length calculation in `ansible-console` when long command inputted
```

This text actually gets inserted into a restructured text document, which means you need to use double backticks in order to get inline code formatting.
```suggestion
  - fix wrong command line length calculation in ``ansible-console`` when long command inputted
```